app/code/Magento/Customer/Test/Mftf/ActionGroup/StorefrontVerifyCustomerOrderDataActionGroup.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="StorefrontVerifyCustomerOrderDataActionGroup">
+        <annotations>
+            <description>Verify a customer's order details on the view order page on the storefront</description>
+        </annotations>
+        <arguments>
+            <argument name="orderId" type="string"/>
+            <argument name="orderNumber" type="string"/>
+            <argument name="createdDate" type="string"/>
+            <argument name="productName" type="string"/>
+            <argument name="grandTotal" type="string"/>
+            <argument name="orderPlacedBy" type="string"/>
+            <argument name="paymentMethod" type="string"/>
+        </arguments>
+        <amOnPage url="sales/order/view/order_id/{{orderId}}" stepKey="goToOrdersPage"/>
```

Comment:
sorry i missed this before

remove this line & create a new actiongroup instead (you always want a separate actiongroup for navigating)

name: `StorefrontGoToCustomerOrderDetailsPageActionGroup`
add description
argument: `<argument name="orderId" type="string"/>`
steps:
* `<amOnPage url="sales/order/view/order_id/{{orderId}}" stepKey="goToOrdersPage"/>`
* waitForPageLoad
* `<waitForText selector="{{StorefrontCustomerAccountMainSection.pageTitle}}" userInput="{{orderNumber}}" stepKey="verifyOrderNo"/>`

keep all below lines in this actiongroup (including `verifyOrderNo`)